script_vts.py

```diff
@@ -18,184 +18,160 @@ def exception_handler(print_exception=False, exception=""):
         del exc_type, exc_obj
         fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
         print("Exception on line: ", exc_tb.tb_lineno, " in ", fname)
-    return
+    return None
 
 
-def checking_arguments(arguments):
+def verify_at_least_one_command_line_argument(arguments):
     """Function checking the number of arguments"""
     if len(arguments) > 1:
         return arguments
-    return
+    return None
+
+
+def parse_arguments():
```

Comment:
Noice!
Can you see the difference this makes for code readability? Especially in your main function.

Reply:
Yes I totally see the difference thanks for the recommendation clearly the code is way better!